solarwinds_apm/trace/otlp_metrics_processor.py

```diff
@@ -42,20 +41,10 @@ def __init__(
         # SW_APM_TRANSACTION_NAME and AWS_LAMBDA_FUNCTION_NAME
         self.env_transaction_name = apm_config.get("transaction_name")
         self.lambda_function_name = apm_config.lambda_function_name
-
-        # TODO Add experimental trace flag, clean up
-        #      https://swicloud.atlassian.net/browse/NH-65067
-        #
-        if not apm_config.get("experimental").get("otel_collector") is True:
-            logger.debug(
-                "Experimental otel_collector flag not configured. Creating meter manager as no-op."
-            )
-            self.apm_meters = NoopMeterManager()
```

Comment:
This is no longer necessary because this processor will not be initialized at all if no `OTEL_METRICS_EXPORTER`. Keeping the no-op class for now in case new config features/refactor later on.